tools/clang-format.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# https://stackoverflow.com/a/16349776
+cd "${0%/*}"
+
+FILES=$(find ../libecho -type f \( -name '*\.cpp' -o -name '*\.h' \))
+
+
+
```

Comment:
I don't like the fact that there are so many new lines :<

Reply:
Fixed